theano/contrib/mkl/mkl_ndarray.h

```diff
@@ -15,12 +15,17 @@
 #define Py_TYPE(o) ((o)->ob_type)
 #endif
 
-#define MAX_NDIM     (16)
-#define MKL_FLOAT32  (11)
-#define MKL_FLOAT64  (12)
-
-char* MKL_TYPE[] = {"", "", "", "int16", "", "int32", "", "int64",
-                    "", "", "", "float32", "float64", ""};
+// MNDA stands for MklNDArray
+#define MNDA_MAX_NDIM   (16)
+#define MNDA_DATA       (0)
+#define MNDA_WORKSPACE  (1)
+#define MNDA_FLOAT32    (11)
+#define MNDA_FLOAT64    (12)
+
+// Placeholder for dtype name. Only support FP32 and FP64 currently.
+// INT8/INT16 will be added to this array in the future.
```

Comment:
only support int8 and int16 in the future, or other int type will also be supported?

If we can't make sure, we can put "other types will be supported in future" or just leave it alone.